app/components/button-group/button-group.tsx

```diff
@@ -71,34 +73,42 @@ export const ButtonGroup: React.FC<ButtonGroupProps> = ({
               }
             }}
             selected={selectedId === props.id}
+            last={props.id === lastButtonId}
           />
         ))}
       {disabled && selectedButton && (
-        <ButtonForButtonGroup {...selectedButton} selected={true} onPress={() => {}} />
+        <ButtonForButtonGroup
+          {...selectedButton}
+          selected={true}
+          onPress={() => {}}
+          last={true}
+        />
       )}
     </View>
   )
 }
 
-const useStyles = makeStyles(({ colors }, selected: boolean) => ({
-  button: {
-    flex: 1,
-    flexDirection: "row",
-    alignItems: "center",
-    justifyContent: "space-between",
-    padding: 10,
-    paddingVertical: 15,
-    marginHorizontal: 3,
-    borderRadius: 5,
-    backgroundColor: selected ? colors.grey4 : colors.grey5,
-  },
-  text: {
-    fontSize: 16,
-    color: selected ? colors.primary : colors.grey1,
-  },
-  buttonGroup: {
-    flexDirection: "row",
-    justifyContent: "space-between",
-    alignItems: "center",
-  },
-}))
+const useStyles = makeStyles(
+  ({ colors }, { selected, last }: { selected: boolean; last: boolean }) => ({
+    button: {
+      flex: 1,
+      flexDirection: "row",
+      alignItems: "center",
+      justifyContent: "space-between",
+      padding: 10,
+      paddingVertical: 14,
+      marginRight: last ? 0 : 10,
```

Comment:
I don't think this is the right way to do it. this problem may be solve in pure css only without having to pass external property.

I'm not an expert in CSS, but I think using some flex that would calculate interval automatically would be better. gpt4 is typically very good at it.